compiler-rt/lib/sanitizer_common/sanitizer_common.h

```diff
@@ -963,19 +957,8 @@ inline void AndroidLogInit() {}
 inline void SetAbortMessage(const char *) {}
 #endif
 
-#if SANITIZER_ANDROID
-void SanitizerInitializeUnwinder();
-AndroidApiLevel AndroidGetApiLevel();
-#else
-inline void AndroidLogWrite(const char *buffer_unused) {}
-inline void SanitizerInitializeUnwinder() {}
-inline AndroidApiLevel AndroidGetApiLevel() { return ANDROID_NOT_ANDROID; }
-#endif
-
 inline uptr GetPthreadDestructorIterations() {
-#if SANITIZER_ANDROID
-  return (AndroidGetApiLevel() == ANDROID_LOLLIPOP_MR1) ? 8 : 4;
-#elif SANITIZER_POSIX
+#if SANITIZER_ANDROID || SANITIZER_POSIX
```

Comment:
does `SANITIZER_POSIX` not cover android?

Reply:
Nope.

```
#define SANITIZER_POSIX                                       \
  (SANITIZER_FREEBSD || SANITIZER_LINUX || SANITIZER_APPLE || \
   SANITIZER_NETBSD || SANITIZER_SOLARIS || SANITIZER_HAIKU)
```

Reply:
Android is Linux

Reply:
@enh-google  Fixed.